src/panels/lovelace/common/compute-card-size.ts

```diff
@@ -4,7 +4,15 @@ export const computeCardSize = (
   card: LovelaceCard | LovelaceHeaderFooter
 ): number | Promise<number> => {
   if (typeof card.getCardSize === "function") {
-    return card.getCardSize();
+    try {
+      const cardSize = card.getCardSize();
+      const timeOut = new Promise((resolve) =>
+        setTimeout(() => resolve(1), 500)
+      );
+      return Promise.race([cardSize, timeOut]) as Promise<number>;
```

Comment:
https://github.com/home-assistant/frontend/blob/dev/src/common/util/promise-timeout.ts

Reply:
Well, I'll be damned if someone hasn't thought of *everything* already...